qiskit_nature/algorithms/excited_states_solvers/excited_states_eigensolver.py

```diff
@@ -18,10 +18,10 @@
 from qiskit.algorithms import Eigensolver
 from qiskit.opflow import PauliSumOp
 
+from qiskit_nature.converters.second_quantization import QubitConverter
 from qiskit_nature.operators.second_quantization import SecondQuantizedOp
-from qiskit_nature.operators.second_quantization.qubit_converter import QubitConverter
-from qiskit_nature.problems.second_quantization.base_problem import BaseProblem
-from qiskit_nature.results.eigenstate_result import EigenstateResult
+from qiskit_nature.problems.second_quantization import BaseProblem
```

Comment:
Just noting this once: I see that you unified the import statements to always import clases from the module they are first exposed in. I do agree with this and am wondering whether we should make this an explicit convention. If we do want to do that, can we enforce this with some linter? However, I cannot think of a solution from the top of my head..

Reply:
Normally this is how we are importing elsewhere where we expose the class/method via the package init file. For sure this is how any tests should be done, since this is how we would normally expect users to do it - it corresponds to the API reference and how we document these.